### PR TITLE
Update netop-manager manifests for notification CRDs

### PR DESCRIPTION
**Pull request by vlella**

### CODE REVIEW

**Comment by tanyatukade** [comment]
Please fix indentation errors in the manifests and run the failing test again

File: provision/acc_provision/templates/cko/0.7.0/netop-manifest-openshift.yaml [hunk 1]
```diff
@@ -787,6 +875,32 @@ rules:
...
+    resources:
+      - notifications
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```
Comment (tanyatukade) [inline provision/acc_provision/templates/cko/0.7.0/netop-manifest-openshift.yaml:889]: Fix indentation error

**Review by tanyatukade — CHANGES_REQUESTED** [review]

File: provision/acc_provision/templates/cko/0.8.0/netop-manifest-openshift.yaml [hunk 2]
```diff
@@ -787,6 +875,32 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+    - notification-ctrlr.netop-types.io
+    resources:
```
Comment (tanyatukade) [inline provision/acc_provision/templates/cko/0.8.0/netop-manifest-openshift.yaml:880]: Fix indentation error

**Review by tanyatukade — APPROVED** [review]
lgtm

**Review by snaiksat — APPROVED** [review]

**Pull request merged**